homeassistant/components/climate/evohome.py

```diff
@@ -0,0 +1,395 @@
+"""Support for Honeywell evohome (EMEA/EU-based systems only).
+
+Support for a temperature control system (TCS, controller) with 0+ heating
+zones (e.g. TRVs, relays) and, optionally, a DHW controller.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.evohome/
+"""
+
+from datetime import datetime, timedelta
+import logging
+from requests.exceptions import HTTPError
```

Comment:
Add a blank line between standard library and 3rd party imports.

Reply:
done